cli/module_generate.go

```diff
@@ -76,7 +76,36 @@ func GenerateModuleAction(cCtx *cli.Context) error {
 }
 
 func (c *viamClient) generateModuleAction(cCtx *cli.Context) error {
-	newModule, err := promptUser()
+	var newModule *moduleInputs
+	var err error
+	resourceSubtype := cCtx.String(moduleFlagResourceSubtype)
+	resourceType := cCtx.String(moduleFlagResourceType)
+	if resourceSubtype != "" {
+		newModule = &moduleInputs{
+			ModuleName:       "my-module",
+			IsPublic:         false,
+			Namespace:        "my-org",
+			Language:         "python",
+			Resource:         resourceSubtype + resourceType,
```

Comment:
Not that it matters since it's unused, but doesn't this require a `space` in between?